UnitsNet/Scripts/Include-GenerateQuantitySourceCode.ps1

```diff
@@ -235,7 +235,19 @@ namespace UnitsNet
 
         public static $quantityName Zero => new $quantityName(0, BaseUnit);
 
-"@; foreach ($unit in $units) {
+"@; 
+    if($quantity.BaseDimensions)
+    {
+@"
+        public static implicit operator BaseDimensions($quantityName quantity)
```

Comment:
Why is this implicit operator useful? I don't see it in your examples above.
I'm generally not too fond of implicit casts as they can trick the programmer up.

Reply:
You can see the example in the following test:

```csharp
[Fact]
public void CheckBaseDimensionMultiplicationWithForceEqualsMassTimesAccelerationByImplicitOverload()
{
    var mass = Mass.FromPounds(205);
    var acceleration = Acceleration.FromMetersPerSecondSquared(9.8);

    BaseDimensions calculatedDimensions = mass * acceleration;
    Assert.True(calculatedDimensions == Force.BaseDimensions);
}
```

These kind of operations are quite useful. I completely understand about implicit casts. I would make them explicit, however it seems WCR doesn't support cast overloads, so I think I'll just remove them entirely and rely on the properties.